device_doctor/lib/src/health.dart

```diff
@@ -43,9 +44,6 @@ Future<HealthCheckResult> closeIosDialog({
     }
     Process proc = await pm.start(command, workingDirectory: dialogDir.path);
     logger.info('Executing: $command');
-    // Discards stdout and stderr as they are too large.
-    await proc.stdout.drain<Object>();
-    await proc.stderr.drain<Object>();
```

Comment:
These two lines block several unit tests. Removing as we are not using this logic at all.

Reply:
+1, I'm always happier if we can remove Stream logic as it's usually over complicated for our use cases